src/pfe/file-watcher/server/test/scripts/exec.sh

```diff
@@ -188,7 +188,7 @@ function setup {
 }
 
 function run {
-    TURBINE_EXEC_TEST_CMD="cd /file-watcher/server; JUNIT_REPORT_PATH=/test_output.xml IMAGE_PUSH_REGISTRY_ADDRESS=${REGISTRY_SECRET_ADDRESS} IMAGE_PUSH_REGISTRY_NAMESPACE=${IMAGE_PUSH_REGISTRY_NAMESPACE} npm run $TEST_SUITE:test:xml"
+    TURBINE_EXEC_TEST_CMD="cd /file-watcher/server; JUNIT_REPORT_PATH=/test_output.xml IMAGE_PUSH_REGISTRY_ADDRESS=${REGISTRY_SECRET_ADDRESS} IMAGE_PUSH_REGISTRY_NAMESPACE=${IMAGE_PUSH_REGISTRY_NAMESPACE} CHE_NS=${NAMESPACE} npm run $TEST_SUITE:test:xml"
```

Comment:
Do we need to have a new env `CHE_NS`? Why not use `NAMESPACE`, just dont want there to be a long list of env just for test...

Reply:
`NAMESPACE` is the variable we call in our test scripts. It doesn't really matter what we set it to be called inside our test because we just set the variable to the value of `NAMESPACE` anyways. If you want I can change it `CHE_NS` -> `NAMESPACE`